lib/smart_app_launch/well_known_capabilities_stu2_test.rb

```diff
@@ -10,7 +10,18 @@ class WellKnownCapabilitiesSTU2Test < Inferno::Test
       If the `sso-openid-connect` capability is supported, then `issuer` and `jwks_uri` must be
       present. If `sso-openid-connect` capability is not supported, then `issuer` must be omitted.
     )
- 
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@372',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@373',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@374',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@381',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@382',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@383',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@384',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@385',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@386',
```

Comment:
This is a `SHOULD` and isn't actually tested here. @386 should be removed.

@386
> [When responding to a `/.well-known/smart-configuration` request the] ...Metadata ...`token_endpoint_auth_methods_supported`[is] OPTIONAL … [and Shall contain the] array of client authentication methods supported by the token endpoint. The options are “client_secret_post”, “client_secret_basic”, and “private_key_jwt”.

Reply:
Removed in [00d865f](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/105/commits/00d865f61fa250db6f0e76c9ea30a521676586a4).